src/server/app/admin/services/users.authentication.server.service.spec.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var _ = require('lodash'),
+	path = require('path'),
+	should = require('should'),
+	q = require('q'),
+
+	deps = require(path.resolve('./src/server/dependencies.js')),
+	config = deps.config,
+
+	userAuthenticationService = require(path.resolve('./src/server/app/admin/services/users.authentication.server.service.js'));
+
+/**
+ * Helpers
+ */
+
+var testDefaultRoles = {
+	'test-role-1': true,
+	'test-role-2': true,
+	'test-role-3': false
+};
+
+function userSpec(key) {
+	return {
+		name: key + ' Name',
+		email: key + '@mail.com',
+		username: key + '_username',
+		password: 'password',
+		provider: 'local',
+		organization: key + ' Organization'
+	};
+}
+
+function validateDefaultRoles(updatedUser) {
+	var keys = _.keys(testDefaultRoles);
+
+	should.exist(updatedUser.roles);
+
+	_.forEach(keys, function(key) {
```

Comment:
We started using arrow functions in the server code everywhere. You should refactor these to use them.